spec/views/items/new.html.erb_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+RSpec.describe 'items/new.html.erb', type: :view do
+
+  it "should include a title field" do
+    view.lookup_context.prefixes = %w[application]
+    stub_template "application/_parent_field.html.erb" => "Parent Info"
+    render
+    expect(rendered).to match /<input class="form-control field-value-input" name="descMetadata\[title\]\[\]" type="text" \/>/
```

Comment:
The expectation feels a little too brittle (breaks if we change css classes, etc.).  I would be inclined to drop the test altogether.